lib/testium-driver-wd.js

```diff
@@ -40,6 +40,17 @@ const debug = require('debug')('testium-driver-wd');
 const _ = require('lodash');
 const wd = require('wd');
 const assert = require('assertive');
+const Gofer = require('gofer');
+const Puppeteer = require('puppeteer');
+const PuppeteerDeviceDescriptors = require('puppeteer/DeviceDescriptors');
+
+// Lighthouse uses node 8 syntax, so we need to make this optional
```

Comment:
this seems to be forced in 3.x version :(

Reply:
One more reason for people to upgrade to node 8+ :)